src/scripts/modules/media/body/body.coffee

```diff
@@ -409,3 +417,10 @@ define (require) ->
 
     resetKeySequence: (e) ->
       key[e.keyCode] = false
+
+    simLink: (evt) ->
+      evt.preventDefault()
+      link = $(evt.currentTarget)
+      @model.set('simUrl', link.attr('href'))
```

Comment:
This design was approved by Kathi in the Pivotal card.

On 04/27/2015 11:20 AM, Philip Schatz wrote:
>
> In src/scripts/modules/media/body/body.coffee 
> <https://github.com/Connexions/webview/pull/1055#discussion_r29162335>:
>
> > @@ -409,3 +417,10 @@ define (require) ->
> >
> >      resetKeySequence: (e) ->
> >        key[e.keyCode] = false
> > +
> > +    simLink: (evt) ->
> > +      evt.preventDefault()
> > +      link = $(evt.currentTarget)
> > +      @model.set('simUrl', link.attr('href'))
>
> Oh, I thought the sims would still be an |<iframe>|. When did 
> |ost-iframe-embeddable| get defined? I don't see it defined anywhere 
> in https://github.com/openstax/tagging-legend/blob/master/README.md
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/Connexions/webview/pull/1055/files#r29162335>.
>
> !DSPAM:1106,553e61e769411793417616!